### PR TITLE
profile updated coloring removed

### LINKED ISSUE #59: Convert the user profile screen to use the nebular colors and styles so we can theme the whole app in the future.
Right now there are lots of additional coloring styles on the user profile page. If we make sure to keep using the nebular UI library we can theme the whole app in the future and easily have features like dark/light themes.

### PR DESCRIPTION
**Pull request by tonae-patterson**
-Line row and input border coloring removed. Also aligned inputs and made all inputs including those on other input pages same style.
- Noticed time calendar was affected by the nav so corrected those styles also.

### CODE REVIEW

File: Frontend/src/app/views/dashboard/time/time.component.scss [hunk 1]
```diff
@@ -138,13 +138,39 @@ button {
...
+    font-size: initial;
+    font-weight: inital;
+    font-family: initial;
+    text-decoration: initial;
+    color:initial;
+  }
+}
+::ng-deep .nb-theme-default nb-popover.nb-overlay-bottom-start .arrow {
+  display: initial;
+}
```
Comment (dyl10s) [inline Frontend/src/app/views/dashboard/time/time.component.scss]: Something weird is going on with the popovers they are all pulled to the left side of the screen.

**Review by dyl10s — CHANGES_REQUESTED** [review]

File: Frontend/src/app/shared/components/create-project/create-project.component.scss [hunk 2]
```diff
@@ -13,6 +14,8 @@
   input, textarea {
     margin-top: 2px;
     margin-bottom: 10px;
+    width: 100%! important;
+    max-width: unset! important;
```
Comment (dyl10s) [inline Frontend/src/app/shared/components/create-project/create-project.component.scss]: What's the point of this? The inputs already have the nebular fullwidth directive.

File: Frontend/src/app/shared/components/create-project/create-project.component.scss [hunk 3]
```diff
@@ -3,6 +3,7 @@
   padding: 20px 20px 20px 20px;
   background-color: white;
   width: min(500px, 95vw);
+  overflow: hidden;
```
Comment (dyl10s) [inline Frontend/src/app/shared/components/create-project/create-project.component.scss:6]: I don't see any problems with this removed. Why did you add this?

File: Frontend/src/app/shared/components/create-project/create-project.component.scss [hunk 4]
```diff
@@ -23,5 +26,6 @@ h4 {
 .actions-footer {
   button {
     margin-right: 10px;
+    margin-top: 10px;
```
Comment (dyl10s) [inline Frontend/src/app/shared/components/create-project/create-project.component.scss:27]: This does not really seem to relate to this story, there seems to be enough margin already. Maybe it looks different on your PC, let me know.

File: Frontend/src/app/views/dashboard/profile/profile.component.scss [hunk 5]
```diff
@@ -1,34 +1,26 @@
+.container {
+  border-radius: 5px;
```
Comment (dyl10s) [inline Frontend/src/app/views/dashboard/profile/profile.component.scss]: Is there a reason for the border-radius here? I don't think the container has a border.

File: Frontend/src/app/views/dashboard/time/time.component.scss [hunk 6]
```diff
@@ -16,7 +16,9 @@ button {
   gap: 10px;
   padding-bottom: 50px;
   justify-content: space-between;
-  h6 {
+  text-align: center;
```
Comment (dyl10s) [inline Frontend/src/app/views/dashboard/time/time.component.scss:19]: I don't see a difference with or without this rule, what did this change?

File: Frontend/src/app/views/dashboard/time/time.component.scss (hunk 1 above)
Comment (tonae-patterson) [reply Frontend/src/app/views/dashboard/time/time.component.scss]: Popover styling has been removed.

File: Frontend/src/app/shared/components/create-project/create-project.component.scss (hunk 2 above)
Comment (tonae-patterson) [reply Frontend/src/app/shared/components/create-project/create-project.component.scss]: So the some of the inputs technically have a 'restricted'  fullwidth. I remember you said we wanted all inputs to be the same length, but some of them although they are fullwidth they have max-width restrictions that wont allow them to be 100% width.

File: Frontend/src/app/shared/components/create-project/create-project.component.scss (hunk 3 above)
Comment (tonae-patterson) [reply Frontend/src/app/shared/components/create-project/create-project.component.scss:6]: I added this overflow property because without it you can stretch the text area beyond the create project component and it did not look good for styling purposes, but If thats a quality we want this can be removed.

File: Frontend/src/app/shared/components/create-project/create-project.component.scss (hunk 4 above)
Comment (tonae-patterson) [reply Frontend/src/app/shared/components/create-project/create-project.component.scss:27]: I added this so that all of the dialog components could have the same styling margins etc and be identical as far as margins.

File: Frontend/src/app/views/dashboard/profile/profile.component.scss (hunk 5 above)
Comment (tonae-patterson) [reply Frontend/src/app/views/dashboard/profile/profile.component.scss]: You were right, unnecessary styling and it has been removed, may have been left over. I do plan to go over all components and ensure there is no left over styling such as this.

File: Frontend/src/app/views/dashboard/time/time.component.scss (hunk 6 above)
Comment (tonae-patterson) [reply Frontend/src/app/views/dashboard/time/time.component.scss:19]: This makes sure the date in mobile view is centered between the arrow buttons.

File: Frontend/src/app/shared/components/create-project/create-project.component.scss (hunk 2 above)
Comment (tonae-patterson) [reply Frontend/src/app/shared/components/create-project/create-project.component.scss]: You actually are correct in the case of this class. This is unnecessary, I will be more observant of redundant variables. May have been transferred over with padding to make sure the dialogs look identical. Removed styling.

**Review by dyl10s — APPROVED** [review]
